### PR TITLE
[FIX] mail: do not toggle reaction on emoji select

### PR DESCRIPTION
**Pull request by oomsveta**

### CODE REVIEW

File: addons/mail/static/src/new/thread/message.js [hunk 1]
```diff
@@ -92,10 +92,9 @@ export class Message extends Component {
                             partners.find(({ id }) => id === this.user.partnerId)
                     );
                     if (reaction) {
-                        this.messaging.removeReaction(reaction);
```
Comment (tsm-odoo) [inline addons/mail/static/src/new/thread/message.js:95]:
```javascript
if (!reaction) {
    this.messaging.addReaction(this.message.id, emoji);
}
```
What about this ? looks easier :)

**Review by tsm-odoo — COMMENTED** [review]
```javascript
if (!reaction) {
    this.messaging.addReaction(this.message.id, emoji);
}
```
What about this ? looks easier :)

**Comment by seb-odoo** [comment]
As usual, it would be nice to have a test to cover it.

**Comment by oomsveta** [comment]
@seb-odoo changes done

**Review by seb-odoo — APPROVED** [review]

**Pull request merged**